package.yaml

```diff
@@ -31,7 +31,7 @@ library:
     - containers >= 0.6.0.1 && < 0.7
     - megaparsec >= 7.0.5 && < 9.3
     - parser-combinators >= 1.1.0 && < 1.4
-    - text >= 1.2.3.1 && < 1.3
+    - text >=1.2.3.1 && <1.3 || >=2.0 && <2.1
```

Comment:
can you just edit 1.3 to 2.1? no need to make a compound range

Reply:
Done and force-pushed.